src/lib/containers/SearchV2.tsx

```diff
@@ -149,7 +150,7 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
       const columnSingleValueQueryFilter: ColumnSingleValueQueryFilter = {
         columnName,
         operator: ColumnSingleValueFilterOperator.LIKE,
-        values: [searchText],
+        values: [`%${searchText}%`],
```

Comment:
This is another bug.  The service expects the wildcard characters to get the response that we expect from the LIKE condition(s)